src/csvcubed/schema/codelist-config/v1_1/schema.json

```diff
@@ -0,0 +1,180 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "id": "https://purl.org/csv-cubed/code-list-config/v1.1",
+    "title": "Schema for a codelist.",
+    "description": "Schema to be used for creation and validation of a single codelist in CSV-W format.",
+    "type": "object",
+    "required": [],
```

Comment:
`title` should be a required field IIRC. I had csvcubed complain at me the other day when I didn't provide a title for an inline code list.